### PR TITLE
Add pusher whiteboard example

### PR DESCRIPTION
**Pull request by coetry**
This PR includes a static vanilla frontend (html, css, js) and a node backend with `pusher` to enable a realtime collaborative experience via managed WebSocket infrastructure.

### CODE REVIEW

**Comment by coetry** [comment]
@leo I renamed to `vanilla-pusher-functions` as per the discussion on slack 😄

File: vanilla-pusher-functions/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Realtime Pusher Whiteboard
+
+This directory is a realtime serverless whiteboard powered by Pusher. The frontend is static vanilla HTML, CSS, and JavaScript and the backend is a single Node function.
```
Comment (msweeneydev) [inline vanilla-pusher-functions/README.md]:
```suggestion
This directory is a realtime serverless whiteboard powered by Pusher. The frontend is static vanilla HTML, CSS, and JavaScript and the backend is a single Node.js function.
```

File: vanilla-pusher-functions/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Creating this example
```
Comment (msweeneydev) [inline vanilla-pusher-functions/README.md]:
```suggestion
## Creating This Example
```

I like this better than 'How we created this example' - thoughts @timothyis ?

File: vanilla-pusher-functions/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+To get started with a realtime whiteboard on Now, you can use the [Now CLI](https://zeit.co/download) to initialize the project:
```
Comment (msweeneydev) [inline vanilla-pusher-functions/README.md]:
```suggestion
To get started with a realtime whiteboard on Now, you can use [Now CLI](https://zeit.co/download) to initialize the project:
```

File: vanilla-pusher-functions/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+```shell
+$ now init vanilla-pusher-functions
+```
+
+## Deploying this example
```
Comment (msweeneydev) [inline vanilla-pusher-functions/README.md]:
```suggestion
## Deploying This Example
```

File: vanilla-pusher-functions/api/push-draw-data.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  CLUSTER: cluster
+} = process.env;
+
+const pusher = new Pusher({
+  appId,
+  key,
+  secret,
+  cluster
+});
+
```
Comment (msweeneydev) [inline vanilla-pusher-functions/api/push-draw-data.js]: What's the idea behind importing the environment variables and assigning them different names?

**Pull request merged**